soc/espressif/Kconfig.sysbuild

```diff
@@ -1,10 +1,144 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
+config BOOTLOADER_MCUBOOT_ESPRESSIF
```

Comment:
this should extend the sysbuild bootloader choice

Reply:
I didn't know it could be extended, thanks! I've changed it as you suggest and removed the `BOOTLOADER_CUSTOM` config I've created before.